### PR TITLE
frontend migrations

### PR DESCRIPTION
**Pull request by aaronchongth**

### CODE REVIEW

File: packages/dashboard/.gitignore [hunk 1]
```diff
@@ -29,3 +29,5 @@ yarn-debug.log*
 yarn-error.log*
```
Comment (koonpeng) [inline packages/dashboard/.gitignore:29]: might as well remove this now, we no longer use yarn.

File: packages/dashboard/src/components/tasks/tasks-app.tsx [hunk 2]
```diff
@@ -191,8 +191,8 @@ export const TasksApp = React.memo(
             filterColumn && filterColumn === 'id_' ? filterValue : undefined,
             filterColumn && filterColumn === 'category' ? filterValue : undefined,
             filterColumn && filterColumn === 'requester' ? filterValue : undefined,
-            filterColumn && filterColumn === 'label=pickup' ? filterValue : undefined,
-            filterColumn && filterColumn === 'label=destination' ? filterValue : undefined,
+            // filterColumn && filterColumn === 'label=pickup' ? filterValue : undefined,
+            // filterColumn && filterColumn === 'label=destination' ? filterValue : undefined,
```
Comment (koonpeng) [inline packages/dashboard/src/components/tasks/tasks-app.tsx:195]: remove them instead of commenting out?

**Review by koonpeng — APPROVED** [review]

File: packages/dashboard/src/components/tasks/tasks-app.tsx (hunk 2 above)
Comment (aaronchongth) [reply packages/dashboard/src/components/tasks/tasks-app.tsx:195]: ah thanks for catching that, I was in the midst of making sure everything built and haven't baked in the labels logic yet, will do so

Comment (aaronchongth) [reply packages/dashboard/src/components/tasks/tasks-app.tsx]: https://github.com/open-rmf/rmf-web/pull/971/commits/a2c44e8182287e1d55e754faba20e9be08023597

**Pull request merged**